samples/financial_transcripts/llm_app.py

```diff
@@ -9,7 +9,7 @@
 
 from chatBot import chatBot
 sys.path.append("../..")   ## add directory above
-from chatbotSkills import count_tokens
+from rag_skills.chatbotSkills import count_tokens
```

Comment:
`count_tokens` does not seem to be skill, in the strict sense of skill, or does it?